python/ray/ml/preprocessors/hasher.py

```diff
@@ -0,0 +1,59 @@
+import collections
+from typing import List
+
+import pandas as pd
+
+from ray.ml.preprocessor import Preprocessor
+
+from ray.ml.preprocessors.utils import simple_hash
+
+
+class FeatureHasher(Preprocessor):
+    """Hash the features of the specified columns.
+
+    The created columns will have names in the format ``hash_{column_names}_{hash}``,
+    e.g. ``hash_column1_column2_0``, ``hash_column1_column2_1``, ...
+
+    Note: Currently sparse matrices are not supported.
+    Therefore, It is recommended to **not** use a large ``num_features``.
+
+    Args:
+        columns: The columns of features that should be projected
+                 onto a single hashed feature vector.
+        num_features: The size of the hashed feature vector.
+    """
+
+    _is_fittable = False
+
+    def __init__(self, columns: List[str], num_features: int):
+        super().__init__()
```

Comment:
I don't think that invoking the parent constructor is required if the parent doesn't have one (i.e. has the default [trivial] constructor).
```suggestion
```